src/main/java/com/williamfeliciano/sba_movies/services/MovieService.java

```diff
@@ -0,0 +1,95 @@
+package com.williamfeliciano.sba_movies.services;
+
+import com.williamfeliciano.sba_movies.dtos.AllMoviesDto;
+import com.williamfeliciano.sba_movies.dtos.MovieDto;
+import com.williamfeliciano.sba_movies.dtos.Rating;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+@Service
+public class MovieService {
```

Comment:
Same comment as for the controller, you're managing many movies, not a single one, the service name should be `MoviesService`.